### DISCUSSION TITLE
so, can i use craft with any framework? like: Next.js?

### DISCUSSION THREAD

**Discussion by leandroruel** [prevwong/craft.js#660] [Q&A]
i'm thinking in use craft instead of a drag and drop i've build in my company. i had no experience with such things and the code now looks ugly, messy and spaghetti, so i don't want to reinvent the wheel, if there are tools out there open source and better i want to use it, so i've take a look at docs and it looks like i can use it to my use case (maybe). im building a page creator, but the pages im creating are mostly reports, table, filter, title, description and buttons to download anything. i've been wondering if i can use Craft with Next.js, is that possible?

**Comment by addlistener** [comment]
Yes it is. The official example uses next.js
